Description: Ignore the dependency on org.jfrog.buildinfo:build-info-extractor-gradle
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/build.gradle
+++ b/build.gradle
@@ -2,9 +2,6 @@
     repositories {
         maven { url 'https://repo.spring.io/plugins-release' }
     }
-    dependencies {
-        classpath 'org.jfrog.buildinfo:build-info-extractor-gradle:2.2.2'
-    }
 }
 
 allprojects {
@@ -15,7 +12,6 @@
 subprojects {
     apply plugin: 'groovy'
     apply plugin: 'maven'
-    apply plugin: 'artifactory'
 
     group = 'org.springframework.build.gradle'
 
@@ -40,28 +36,6 @@
     }
 }
 
-artifactory {
-    contextUrl = 'https://repo.spring.io'
-    publish {
-        repository {
-            repoKey = 'plugins-snapshot-local'
-            username = repoUser
-            password = repoPass
-        }
-        defaults {
-            publishConfigs('archives')
-        }
-    }
-    resolve {
-        repository {
-            repoKey = 'libs-releases'
-            username = repoUser
-            password = repoPass
-            maven = true
-        }
-    }
-}
-
 task wrapper(type: Wrapper) {
     description = 'Generates gradlew[.bat] scripts'
     gradleVersion = '1.11'
